src/braket/devices/device.py

```diff
@@ -33,7 +33,12 @@ def __init__(self, name: str, status: str):
 
     @abstractmethod
     def run(
-        self, task_specification: Union[Circuit, Problem], shots: Optional[int], *args, **kwargs
+        self,
+        task_specification: Union[Circuit, Problem],
+        shots: Optional[int],
+        inputs: Optional[Dict[str, float]],
```

Comment:
Do we want to default here too?
```suggestion
        inputs: Optional[Dict[str, float]] = None,
```

Reply:
Ya then we should probably do the same for shots